wgpu-core/src/device/mod.rs

```diff
@@ -3598,10 +3598,7 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
         A::hub(self).buffers.label_for_resource(id)
     }
 
-    pub fn buffer_destroy<A: HalApi>(
-        &self,
-        buffer_id: id::BufferId,
-    ) -> Result<(), resource::DestroyError> {
```

Comment:
nitpick: It'd be nice to have a separate commit for moving `DestroyError` to a direct `use`. This would separate some mechanical changes from the more substantive changes in the `remove open field from CommandBuffer` commit.

Reply:
I'll probably undo the import since every other type in that module has the qualified error type.